portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,38 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private ArrayList<String> messages;
+
+    public ArrayList<String> getMessages(){
+        ArrayList<String> messages = new ArrayList<String>();
+        messages.add ("Hi");
+        messages.add ("Hey, How's it going?");
+        messages.add ("Good");
+        return messages;
+    }
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Edem!");
+    //Get messages
```

Comment:
The amount of commenting is sometimes a personal preference and sometimes teams will come up with team guidelines.  In general, developers probably under-comment rather than over-comment, so if you are going to lean one way from perfection, leaning towards over-commenting is not a bad thing.  In this particular case I would probably remove the comments since the code itself seems just as readable as the comments.

Reply:
Makes sense. Thanks Kevin.